generator/mavschema.xsd

```diff
@@ -202,7 +202,16 @@
     </xs:complexType>
 </xs:element>
 
-<xs:element name="deprecated">
+<xs:element name="wip">
```

Comment:
FYI, just a reorder for wip and deprecate, which is what makes this a bit messier. Sorry.